tests/scripts/save_policy_to_safetensor.py

```diff
@@ -103,8 +103,14 @@ def save_policy_to_safetensors(output_dir, env_name, policy_name, extra_override
 
 
 if __name__ == "__main__":
-    # Instructions: include the policies that you want to save artifacts for here. Please make sure to revert
-    # your changes when you are done.
-    env_policies = []
+    env_policies = [
```

Comment:
IMO we should leave this blank (as per my deleted comment above). It makes sure that you have to be 100% explicit about which artifacts you want to update. For example, maybe you made changes that should only affect 1 policy, but inadvertently you affected them all.

Reply:
@alexander-soare Added some messages:

<img width="1002" alt="Screenshot 2024-05-20 at 12 08 37" src="https://github.com/huggingface/lerobot/assets/4681518/2e31d55e-08da-4897-802b-551b1d099e6c">